nvflare/fuel/utils/omegaconf_loader.py

```diff
@@ -11,19 +11,43 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from typing import Dict, Optional
+
+from omegaconf import DictConfig, OmegaConf
```

Comment:
We may consider package this omegaconf_loader.py separately from the "main" NVFlare. This adds another additional third party dependency to NVFlare, which is a down side. This kind of "extra" to NVFlare could be endless.

Reply:
We can discuss about this at team meeting.

As to how to add to optional dependencies, we add the code to "app_opt" (so they are not FUEL).
And we can either change one item or add a new entry in setup.cfg: https://github.com/NVIDIA/NVFlare/blob/dev/setup.cfg#L37